### PR TITLE
#58 help button added

### PR DESCRIPTION
**Pull request by mohsinshabir1234**
Added a button on bottom right to reach discord help station.

### CODE REVIEW

File: gui/index.html [hunk 1]
```diff
@@ -44,9 +44,29 @@ <h1 class="heading">
...
   <script src="control.js"></script>
+  <script>
+    const {
+      shell
+    } = require('electron');
+
+    function openLinkInDefaultBrowser() {
+      // Replace the URL with the link you want to open
+      shell.openExternal(' https://definex.in/discord');
+    }
```
Comment (adsau59) [inline gui/index.html:68]: dont keep this in html, move it into control.js

File: gui/index.html [hunk 2]
```diff
@@ -44,9 +44,29 @@ <h1 class="heading">
   <!-- toast -->
   <div id="toast-container" class="toast-container position-fixed bottom-0 end-0 p-3" style="z-index:99999999999">></div>
 
+ 
+  
+    
+    <div class="help-button">
+       <button type="button" class="btn btn-primary" onclick="openLinkInDefaultBrowser()">
+    +
```
Comment (adsau59) [inline gui/index.html:52]: instead use a help icon from font awesome

File: gui/index.html (hunk 1 above)
Comment (adsau59) [reply gui/index.html:68]: also remove space from the link

**Review by adsau59 — CHANGES_REQUESTED** [review]
changes requested

**Comment by mohsinshabir1234** [comment]
#58

**Comment by mohsinshabir1234** [comment]
#58

**Review by adsau59 — APPROVED** [review]

**Comment by mohsinshabir1234** [comment]
Thanks

**Pull request merged**